Automattic/wp-calypso
# Domains: Remove unused TLD validation schemas

#### Changes proposed in this Pull Request

While looking around the domains code, I noticed that we're not using the TLD validation schemas at all. The `WithContactDetailsValidation` HoC was used only for the UK form, and even in it, we don't use any of the props that the HoC provides. That means that it can be removed. There are no other usages of the related query component either, which consequently means that the `getValidationSchemas` selector isn't used. That means the entire state tree can also be removed. The PR removes all of that unused code.

Part of #58453.

#### Testing instructions

* Verify the forms for purchasing a UK and a FR domain still validate correctly.
* Verify none of the removed code is not used.
* Verify all tests pass.
